src/styles/index.css

```diff
@@ -0,0 +1,61 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: grey;
+}
+
+.product-card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+}
+
+.product-card__image {
+  background: center url(../images/imac.jpeg);
+  background-size: cover;
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px;
+}
+
+.product-card__title {
+  width: 166px;
+  height: 36px;
+}
+
+.product-card__name {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.product-card__code {
+  line-height: 14px;
+  font-size: 10px;
+  color: #616070;
+  font-weight: 400;
+  display: flex;
+  text-align: left;
+}
+
+.product-card__buy-btn {
+  display: inline-block;
+  width: 166px;
+  height: 40px;
+  text-align: center;
+  text-decoration: none;
+  background-color: #00acdc;
```

Comment:
The color value for the background-color property should be in lowercase to adhere to the code style rules.